src/OrbitGl/AccessibleInterfaceProvider.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_ACCESSIBLE_INTERFACE_PROVIDER_H_
+#define ORBIT_GL_ACCESSIBLE_INTERFACE_PROVIDER_H_
+
+#include "OrbitAccessibility/AccessibleInterface.h"
+
+namespace orbit_gl {
+
+/* Base class to provide accessibility in the capture window. */
+class AccessibleInterfaceProvider {
+ public:
+  explicit AccessibleInterfaceProvider(AccessibleInterfaceProvider* parent) : parent_(parent) {}
+
+  [[nodiscard]] orbit_accessibility::AccessibleInterface* GetOrCreateAccessibleInterface();
+  [[nodiscard]] const orbit_accessibility::AccessibleInterface* GetAccessibleInterface() const {
+    return accessibility_.get();
+  }
+
+  [[nodiscard]] virtual AccessibleInterfaceProvider* GetParent() const { return parent_; }
+
+ private:
+  AccessibleInterfaceProvider* parent_;
```

Comment:
Same for this - it's not used, you even declare a variable of the same name in `CaptureViewElement` as type `CaptureViewElement*`, and an additional `AccessibleInterfaceProvider*` in `TimeGraph`. I'd just remove the declaration here, it shouldn't be needed. This will create chaos from multiple inheritance otherwise.

Reply:
Again, you are right, thanks.